Makefile

```diff
@@ -77,6 +77,11 @@ integration-kind:
 	KUBECONFIG="$(kind get kubeconfig-path --name="integration")" go test -mod vendor -p $(GO_PARALLEL_JOBS) -tags 'integration' -timeout 700s ./integration -count 1
 	kind delete cluster --name=integration
 
+# Run the extension integration tests, assuming the extensions
```

Comment:
I'm not sure how GOPATH is involved. It looks like this is running them from /tmp?

Reply:
ooh, good catch, fixed